engine/src/main/java/com/datatorrent/stram/StramClient.java

```diff
@@ -448,6 +450,14 @@ public void startApplication() throws YarnException, IOException
       }
       String libJarsCsv = copyFromLocal(fs, appPath, localJarFiles.toArray(new String[]{}));
 
+      if (UserGroupInformation.isSecurityEnabled() && customKeystoreConfig != null && customKeystorePath != null) {
+        String[] customSSLConfArr = {customKeystoreConfig, customKeystorePath};
```

Comment:
The keystore file is supposed to be same location as the keystore config file. They are both added as local resources and if you see how `copyFromLocal` works it returns a comma separated list in customSSLFileConf which is then supplied as the second argument to LaunchContainerRunnable.addFilesToLocalResources. Ultimately on the Stram side this is treated as a resource name by WebAppsUtil for loading and this local resource is already made available in the right location via this PR.